assets/src/components/circleButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactNode } from "react"
+import { joinClasses } from "../helpers/dom"
+
+export const CircleButton = ({
+  children,
+  onClick,
+  isActive,
+  title,
+}: {
+  children?: ReactNode
+  onClick?: () => void
+  isActive: boolean
+  title?: string
+}) => (
```

Comment:
Should this take arguments to specify its size? Either `small`/`medium`/`large` or `height`/`width`? Or is there a way to do that that I missed?

Reply:
From the Figma, there are numerous sizes of the user avatar, but only the one size of the circle button. After discussion with design I decided to make the avatar fill its container and make the button a fixed size.